plugins/modules/inventory_workflow_manager.py

```diff
@@ -1799,7 +1799,7 @@ def provision_wired_device_v1(self, device_ip, site_name, device_type):
             site_name (str): The name of the site where the device will be provisioned.
```

Comment:
can we change the site_name to site_name_hierarchy? You can also change in provision_wired_device_v2...

Reply:
updated the code